test_app/settings_test.py

```diff
@@ -60,3 +60,5 @@
     ('admin1', 'registration_admin1@mail.server.com'),
     ('admin2', 'registration_admin2@mail.server.com'),
 )
+
+ACCOUNT_ACTIVATION_DAYS = 7
```

Comment:
this shouldn't be necessary since we use `override_settings` on the test cases. why is this change necessary? are the overrides no longer working?

Reply:
The tests did not pass on a fresh fork from master, before I made any changes (fresh clone). 
Adding the `ACCOUNT_ACTIVATION_DAYS` to `settings_test.py` helped. 
I did not investigate it further, though. I assumed that it was  accidentally removed.
Let's look into it!
Could you check the results of `invoke test` on master?

Reply:
seems to be a weird interaction between pytest/pytest-django and our setup in recent versions.

Reply:
If I run tests without `ACCOUNT_ACTIVATION_DAYS` in the `settings_test.py` 5 tests from `default_backend.DefaultBackendViewTests` fail (only from that file). 
If I then apply `@override_settings(ACCOUNTS_ACTIVATION_DAYS=7)` to those (failing) tests, they pass.
This result suggests that there is some problem with `@override_setting` when used jointly at class and method/test level.

Without going any deeper, one way to resolve the issue is to add the missing line to those tests. We then can safely remove `ACCOUNT_ACTIVATION_DAYS` from the `settings_test.py`.
Any preference or suggestion?

Reply:
i'm ok with leaving the setting in `settings_test.py`, thanks for the patch!

Reply:
Great! Thanks!